apps/react-app/src/pods/screen2/components/screen2.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const Screen2: React.FC = () => {
+  return <h1>Hello from Screen 2</h1>;
+};
```

Comment:
You can rename this to the following

src/pods/choose-template/choose-template.container.tsx

And include a barrel index